frontend/src/app/firewood/forests/buy-firewood-permit/buy-firewood-permit.component.html

```diff
@@ -8,7 +8,7 @@ <h1 id="skip-nav-target" tabindex="-1">
       <span>Buy a firewood permit</span>
     </h1>
     <div *ngIf="!showRules">
-      <h2 *ngIf="forest">You are purchasing a...</h2>
+      <h2 *ngIf="forest">You are purchasing a... {{forest.minCost}}</h2>
```

Comment:
Should the min cost be here or is this leftover from testing/debugging?

Reply:
@mgwalker - To be completely honest. I 100% agree with you. I'd prefer that the value was computed. I'm sure it's just my general noobishness when it comes to Angular, but I haven't, up to this point been able to make it work. I keep getting a NaN value or nothing at all when I do the computation on the .ts file.

Reply:
Gotcha. In the name of keeping moving, then, let's do it this way and maybe sometime in the future someone can change it over. 😄

Reply:
@mgwalker - that is leftover. I'll remove it. Gratsi!